### PR TITLE
Include summary stats at the end of the GH issue

### PR DESCRIPTION
**Pull request by amcasey**
...so that we know if the number of repos we're successfully analyzing drops (which it tends to do as the package manager go in and out of favor).

### CODE REVIEW

File: src/main.ts [hunk 1]
```diff
@@ -287,13 +305,42 @@ export async function mainAsync(params: GitParams | UserParams): Promise<GitResu
         await execAsync(processCwd, "rm -rf " + newTscDirPath);
     }
 
+    console.log("Statuses");
+    statusCounts.forEach((count, status) => console.log(`${status}\t${count}`));
```
Comment (DanielRosenwasser) [inline src/main.ts:309]: Amazing how much this looks like an error at a glance.

File: src/main.ts [hunk 2]
```diff
@@ -287,13 +305,42 @@ export async function mainAsync(params: GitParams | UserParams): Promise<GitResu
...
+        statusCounts.forEach((count, status) => {
+            totalCount += count;
+            switch (status) {
+                case "NewBuildSucceeded":
+                case "NewBuildFailed":
+                case "NewBuildHadErrors":
+                    analyzedCount += count;
+                    break;
+            }
+        });
```
Comment (DanielRosenwasser) [inline src/main.ts:323]:
```suggestion
        totalCount += statusCounts.get("NewBuildSucceeded") ?? 0;
        totalCount += statusCounts.get("NewBuildFailed") ?? 0;
        totalCount += statusCounts.get("NewBuildHadErrors") ?? 0;
```

**Review by DanielRosenwasser — APPROVED** [review]

File: src/main.ts (hunk 2 above)
Comment (amcasey) [reply src/main.ts:323]: What happened to `analyzedCount`?

Comment (DanielRosenwasser) [reply src/main.ts:323]: 🤦‍♂️ misread, sorry

**Pull request merged**